packages/client/src/library/components/InputAddress.js

```diff
@@ -4,7 +4,14 @@ import { useAddressValidation } from "hooks";
 import Svg from "library/Svg";
 import { isAddr, formatAddress } from "utils";
 
-const InputAddress = ({ web3, value, isValid, onChange }) => {
+const InputAddress = ({ 
+  web3, 
```

Comment:
Do we still need web3 as a prop when we expect isValid and onChange to handle the validation?